### ISSUE TITLE
automake-1.15 is required, fails to build on CentOS-7

### ISSUE THREAD

**Issue by nixpanic** [gluster/glusterfs-coreutils#19]
I'm adding a test in the [CentOS CI](https://ci.centos.org/view/Gluster/job/gluster_coreutils/) for verification that upstream Gluster patches did not break glusterfs-coreutils. The test-case is based on a script that @anoopcs9 provided: https://github.com/nixpanic/glusterfs-patch-acceptance-tests/blob/centos-ci/glusterfs-coreutils/centos-ci/glusterfs-coreutils/run-test.sh

The script [fails](https://ci.centos.org/view/Gluster/job/gluster_coreutils/2/console) with:

```
configure.ac:28: the top level
configure.ac:13: error: require Automake 1.15, but have 1.13.4
```

Could the sources be modified to build on CentOS-7?

**Comment by craigcabrey** [comment]
I don't see why not, we can change it to 1.13.2 to avoid potential regressions introduced in 1.13 per [this page](https://autotools.io/forwardporting/automake.html).

Does that sound reasonable?

**Comment by anoopcs9** [comment]
@craigcabrey , Will you be patching it by yourself?
@nixpanic , Thanks for reporting it asap.

**Comment by craigcabrey** [comment]
Changed in [`c155815`](https://github.com/gluster/glusterfs-coreutils/commit/c155815073667968aade0880abaecf709c5416fc).

**Comment by nixpanic** [comment]
Yes, that should work just fine on CentOS-7. I have not tried it out yet
with a local change. Let me (or Anoop) know if you need us to test it.

Thanks!

**Comment by craigcabrey** [comment]
I tested it on my local machine and there were no problems.

I also tagged an initial releaase (long overdue imo). Tagged at `0.2.0` because I didn't want to conflict with the version I released internally at Facebook (`0.1.0`).

**Comment by anoopcs9** [comment]
@craigcabrey , Great.. I was about to ask you on tagging a release. Thanks for fixing the automake version dependency so quick. 

@nixpanic , You are all set to try another test run.

And its time for me to update fedora packages...Cool.
